docs/source/azure/examples/vmwithpublicip.rst

```diff
@@ -0,0 +1,13 @@
+Find Virtual Machines with Public IP address
+============================================
+
+.. code-block:: yaml
+
+     policies:
+       - name: vms-with-public-ip
+         resource: azure.vm
+         filters:
+          - type: network-interface
+            key: 'properties.ipConfigurations[].properties.publicIPAddress.id'
+            op: eq

```

Comment:
See not-null examples, can remove op entirely
http://capitalone.github.io/cloud-custodian/docs/filters.html